platform/macos/SCsub

```diff
@@ -30,6 +33,12 @@ files = [
     "gl_manager_macos_legacy.mm",
 ]
 
+if env.editor_build:
+    files += [
+        "embedded_game_view_plugin.mm",
+        "embedded_process_macos.mm",
+    ]
```

Comment:
I'd suggest putting editor specific files in an `editor` subfolder, like we do for modules, to keep a clear distinction between runtime and editor components.

Reply:
Very good – will do that 👍🏻